src/frequenz/quantities/_apparent_power.py

```diff
@@ -0,0 +1,251 @@
+
+        Returns:
+            A new apparent power quantity.
+        """
+        return cls._new(mva, exponent=-3)
+
+    @classmethod
+    def from_kilo_volt_amperes(cls, kva: float) -> Self:
+        """Initialize a new apparent power quantity.
+
+        Args:
+            kva: The apparent power in kilovolt-amperes (kVA).
+
+        Returns:
+            A new apparent power quantity.
+        """
+        return cls._new(kva, exponent=3)
+
+    @classmethod
+    def from_mega_volt_amperes(cls, mva: float) -> Self:
+        """Initialize a new apparent power quantity.
+
+        Args:
+            mva: The apparent power in megavolt-amperes (MVA).
+
+        Returns:
+            A new apparent power quantity.
+        """
+        return cls._new(mva, exponent=6)
+
+    def as_volt_amperes(self) -> float:
+        """Return the apparent power in volt-amperes (VA).
+
+        Returns:
+            The apparent power in volt-amperes (VA).
+        """
+        return self._base_value
+
+    def as_milli_volt_amperes(self) -> float:
+        """Return the apparent power in millivolt-amperes (mVA).
+
+        Returns:
+            The apparent power in millivolt-amperes (mVA).
+        """
+        return self._base_value * 1e3
+
+    def as_kilo_volt_amperes(self) -> float:
+        """Return the apparent power in kilovolt-amperes (kVA).
+
+        Returns:
+            The apparent power in kilovolt-amperes (kVA).
+        """
+        return self._base_value / 1e3
+
+    def as_mega_volt_amperes(self) -> float:
+        """Return the apparent power in megavolt-amperes (MVA).
+
+        Returns:
+            The apparent power in megavolt-amperes (MVA).
+        """
+        return self._base_value / 1e6
+
+    # We need the ignore here because otherwise mypy will give this error:
+    # > Overloaded operator methods can't have wider argument types in overrides
+    # The problem seems to be when the other type implements an **incompatible**
+    # __rmul__ method, which is not the case here, so we should be safe.
+    # Please see this example:
+    # https://github.com/python/mypy/blob/c26f1297d4f19d2d1124a30efc97caebb8c28616/test-data/unit/check-overloading.test#L4738C1-L4769C55
+    # And a discussion in a mypy issue here:
+    # https://github.com/python/mypy/issues/4985#issuecomment-389692396
+    @overload  # type: ignore[override]
+    def __mul__(self, scalar: float, /) -> Self:
+        """Scale this power by a scalar.
+
+        Args:
+            scalar: The scalar by which to scale this power.
+
+        Returns:
+            The scaled power.
+        """
+
+    @overload
+    def __mul__(self, percent: Percentage, /) -> Self:
+        """Scale this power by a percentage.
+
+        Args:
+            percent: The percentage by which to scale this power.
+
+        Returns:
+            The scaled power.
+        """
+
+    @overload
+    def __mul__(self, other: timedelta, /) -> Energy:
```

Comment:
These need to go as well, in both apparent power and reactive power. Later, if we add `ReactiveEnergy`, etc., we can add these back.

Reply:
Right, done